buildSrc/DesktopBuilder.js

```diff
@@ -22,7 +22,7 @@ function build(dirname, version, targets, updateUrl, nameSuffix, notarize, outDi
 		version,
 		updateUrl,
 		path.join(dirname, "/resources/desktop-icons/logo-solo-red.png"),
-		nameSuffix !== "-snapshot",
+		nameSuffix !== '-snapshot' && updateUrl !== "", // don't sign if it's a test build or if we don't download updates
```

Comment:
Doesn't it make sense to sign even if no updates are expected? Especially for mac/windows builds, they must still be distributed more-or-less packaged. Or is it a different signature?

Reply:
I'd argue to let people that need signatures on their installers decide how they want to handle that. The app itself doesn't care about its signature after it is installed.

Of course it would be possible to make the update process configurable for custom builds so people/institutions can use that functionality to update their builds. Then the signatures would make sense.

Afaict the windows installer and unpacked build works just fine without signature, but I can't test the mac build.

Reply:
mac would be more difficult but it's not one of our goals right now, you are right. Maybe the right way is to have a command line flag but it's not really needed, especially for custom builds.
Let's leave it like it is then.